src/server/game/Entities/Player/Player.cpp

```diff
@@ -4507,13 +4507,14 @@ void Player::BuildPlayerRepop()
 {
     WorldPackets::Misc::PreRessurect packet;
     packet.PlayerGUID = GetGUID();
-    GetSession()->SendPacket(packet.Write());
+    SendMessageToSet(packet.Write(), true);
```

Comment:
This seems like a correct change to me (after all this packet handler is responsible for clearing in-flight spell missiles) but this belongs in a separate PR

Reply:
true. i removed that part